### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by pranavbharadwaj007**
Capture

### CODE REVIEW

**Comment by niranjanhegde144** [comment]
Change the NavBar Color

**Comment by akramkazmi71** [comment]
In order to change the NavBar color to dark version, go to App.js 
`<Navbar bg="light" collapseOnSelect={true} varient="light" expand="lg">`

Change b=="dark" and varient="dark" that would fix this.

**Comment by akramkazmi71** [comment]
Also, @pranavbharadwaj007 you need not put the image everytime. Netlify has a feature which creates a temp deployment using the current PR and we can view the same.

File: src/App.css [hunk 1]
```diff
@@ -74,8 +83,79 @@
...
+  font-family: 'Fredoka One', cursive;
+  font-size: 50px;
+}
+
+.text-small {
+  font-family: 'Fredoka One', cursive;
+  font-size: 19px;
+}
+
+@media only screen and (max-width: 1178px) {
```
Comment (akramkazmi71) [inline src/App.css:140]: set max-width as "960px", lets keep it similar everywhere.

File: src/components/contributors.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ const fontAwesomeStyle = {
 
 let mapDetail = (detail) => {
   return (
+    
+
+
```
Comment (akramkazmi71) [inline src/components/contributors.js:14]: why so many blank lines?

File: src/App.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { Nav, Navbar } from "react-bootstrap";
 function App() {
   return (
     <div>
-      <Navbar bg="light" collapseOnSelect={true} varient="light" expand="lg">
+      <Navbar bg="dark" collapseOnSelect={true} varient="dark" expand="lg">
         <Navbar.Brand as={Link} to="/">
           <img
             src={process.env.PUBLIC_URL + "/favicon.ico"}
```
Comment (akramkazmi71) [inline src/App.js:21]: Need to update this as well. Image is not visible.

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Comment by akramkazmi71** [comment]
Also, @pranavbharadwaj007 lets have the same theme all over the website. I can see only the home page and the result page have a similar theme. Lets do that for other pages as well.


Also, there will be a merge conflict later on. So do update your forked repo and then make a commit.

File: src/App.js (hunk 3 above)
Comment (niranjanhegde144) [reply src/App.js:21]: Make sure the logo color is turned to white. As it is not visible.

**Pull request closed without merging**